src/components/Navigation/NavigationItems.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import NavigationItem from './NavigationItem';
+import Anchor from '../Anchor';
+
+export const Container = styled.div.withConfig({ displayName: 'NavigationItemsContainer' })`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+
+  background: ${({ theme }) => theme.tab.bg};
+  color: ${({ theme }) => theme.tab.fg};
+  font-size: ${({ theme }) => theme.tab.fontSize};
+  margin-bottom: ${({ theme }) => theme.tab.marginBottom};
+
+  & > * {
+    margin: ${({ theme }) => theme.tab.margin};
+  }
+
+  & > *:last-of-type {
+    margin: 0;
+    padding-right: 0;
+  }
+
+  & > a {
+    color: inherit;
+    text-decoration: none;
+  }
+`;
+
+export const linksPropType = PropTypes.arrayOf(PropTypes.shape({
+  to: PropTypes.string,
+  onClick: PropTypes.func,
+  content: PropTypes.node.isRequired,
+  exact: PropTypes.bool,
+}));
+
+const NavigationItems = ({ links }) => (
+  <Container>
+    {links.map((link) => (
+      <Anchor key={`${link.content}_${link.to}`} to={link.to} onClick={link.onClick} exact={link.exact}>
```

Comment:
Since the `link.to` property is not required it seems like it would be possible to have duplicate keys if you receive multiple links with the same content value and no to value (could be useful if you wanted to create a set of navigation item links that function as callbacks that don't have an actual destination).

Same situation with multiple links with the same to value (but that might be a little odd, can't think of a reason to do that intentionally).